test/polygon/aave/aaveV3-import-test.ts

```diff
@@ -0,0 +1,368 @@
+    {
+      inputs: [
+        {
+          internalType: "address",
+          name: "asset",
+          type: "address"
+        },
+        {
+          internalType: "uint256",
+          name: "amount",
+          type: "uint256"
+        },
+        {
+          internalType: "address",
+          name: "onBehalfOf",
+          type: "address"
+        },
+        {
+          internalType: "uint16",
+          name: "referralCode",
+          type: "uint16"
+        }
+      ],
+      name: "deposit",
+      outputs: [],
+      stateMutability: "nonpayable",
+      type: "function"
+    },
+    {
+      inputs: [
+        {
+          internalType: "address",
+          name: "asset",
+          type: "address"
+        },
+        {
+          internalType: "uint256",
+          name: "amount",
+          type: "uint256"
+        },
+        {
+          internalType: "address",
+          name: "onBehalfOf",
+          type: "address"
+        },
+        {
+          internalType: "uint16",
+          name: "referralCode",
+          type: "uint16"
+        }
+      ],
+      name: "supply",
+      outputs: [],
+      stateMutability: "nonpayable",
+      type: "function"
+    }
+  ];
+
+  let dsaWallet0: any;
+  let masterSigner: Signer;
+  let instaConnectorsV2: Contract;
+  let connector: any;
+
+  const wallet = ethers.Wallet.fromMnemonic(mnemonic);
+
+  before(async () => {
+    await hre.network.provider.request({
+      method: "hardhat_reset",
+      params: [
+        {
+          forking: {
+            //@ts-ignore
+            jsonRpcUrl: hre.config.networks.hardhat.forking.url,
+            blockNumber: 26652016
+          }
+        }
+      ]
+    });
+    masterSigner = await getMasterSigner();
+    [wallet0] = await ethers.getSigners();
+    await hre.network.provider.send("hardhat_setBalance", [account, ethers.utils.parseEther("10").toHexString()]);
+
+    await hre.network.provider.request({
+      method: "hardhat_impersonateAccount",
+      params: [account]
+    });
+
+    signer = await ethers.getSigner(account);
+
+    await token.connect(signer).transfer(wallet0.address, ethers.utils.parseEther("8"));
+
+    instaConnectorsV2 = await ethers.getContractAt(abis.core.connectorsV2, addresses.core.connectorsV2);
+    connector = await deployAndEnableConnector({
+      connectorName,
+      contractArtifact: ConnectV2AaveV3ImportPermitPolygon__factory,
+      signer: masterSigner,
+      connectors: instaConnectorsV2
+    });
+
+    const aave = new ethers.Contract(aaveAddress, aaveAbi);
+
+    // approve DAI to aavePool
+    await token.connect(wallet0).approve(aaveAddress, parseEther("8"));
+
+    //deposit DAI in aave
+    await aave.connect(wallet0).supply(DAI, parseEther("8"), wallet.address, 3228);
+    console.log("Supplied DAI on aave");
+
+    //borrow USDC from aave
+    await aave.connect(wallet0).borrow(USDC, parseUnits("5", 6), 2, 3228, wallet.address);
+    console.log("Borrowed USDC from aave");
```

Comment:
@Richa-iitr Move this to another `it` hook? Like "Should create Aave v3 position of DAI(collateral) and USDC(debt)"